fetch.sh

```diff
@@ -10,4 +10,8 @@ dot="$(cd "$(dirname "$([ -L "$0" ] && $READLINK_CMD -f "$0" || echo "$0")")"; p
 
 source $dot/radar-base.sh
 
-fetch;
+if [[ -z "$1" ]]; then
```

Comment:
Rather than checking $1 you could use $@ (which contains all arguments passed to this script).